src/components/common/Modal/styled.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import { device } from 'Themes';
+import ReactModalAdapter from './modal';
+
+export const StyledModal = styled(ReactModalAdapter)`
+  &__overlay {
```

Comment:
Mentioned it in a previous PR, ideally you wouldn't be using classNames for these. Styled components is kinda trying to follow the React components. Small components that you break up and reuse.

Reply:
@girwin90 These classNames were not created but came with the react-modal library that I am using. I need to amend the styling and position of the modal which is why I've referenced these classes here.